### ISSUE TITLE
Broken with the pry 0.13.0

### ISSUE THREAD

**Issue by lastobelus** [guard/guard#953]
```
/Users/lasto/.rvm/gems/ruby-2.7.0/gems/guard-2.16.1/lib/guard/jobs/pry_wrapper.rb:133:in `_setup': undefined method `file=' for #<Pry::History:0x00007fabc0258988>
```

this is because pry have refactored Pry.config.history to be a firstclass object.

EDIT:
Instead of setting `Pry.config.history.file`, should set `Pry.config.history_file`

**Comment by georgeu2000** [comment]
Yes, I ran into this too.  Worked for me with Pry 0.11.3.

**Comment by yubele** [comment]
I worked pry 0.12.2.

This can be done with the following modifications.
https://github.com/newsdict/newsdict.io/pull/48/commits/011840d1016feb3abd87ed8fb38766d4b47804ca

**Comment by rymai** [comment]
There seem to be other issues (e.g. https://travis-ci.org/github/guard/guard/jobs/665304043?utm_source=github_status). I'll look into fixing those as well.

**Comment by IanVaughan** [comment]
I am seeing this now we upgraded `pry-byebug` which upgraded pry from `0.12.2` to `0.13.0`
```
guard (2.16.1)
guard-compat (1.2.1)
guard-rspec (4.7.3)
guard-rubocop (1.3.0)

pry (0.13.0, 0.12.2)
pry-byebug (3.9.0, 3.8.0, 3.7.0)
pry-rails (0.3.9)
```

**Comment by rymai** [comment]
#958 hopefully fixes the Pry 0.13.0 compatibility issues. I've released Guard `2.16.2`. Let me know how it goes. 🙏
